include/IrcUtil/irclagtimer_p.h

```diff
@@ -60,6 +60,7 @@ class IrcLagTimerPrivate : public QObject,  public IrcMessageFilter
     IrcConnection* connection;
     QTimer timer;
     int interval;
+    int pendingPings;
```

Comment:
could you initialize pendingPings to 0 in IrcLagTimerPrivate() initializer list?